src/seedu/addressbook/Main.java

```diff
@@ -111,7 +112,10 @@ private CommandResult executeCommand(Command command)  {
             CommandResult result = command.execute();
             storage.save(addressBook);
             return result;
-        } catch (Exception e) {
+        }catch (StorageWriteException swe) {
+            return new CommandResult(MESSAGE_STORAGE_ERROR);
```

Comment:
In general, just showing the error message or printing the stack trace is not considered "good exception handling".
Think how you can improve the user experience in light of the exception that just occurred.